.github/workflows/linux-build.yaml

```diff
@@ -0,0 +1,43 @@
+name: Linux Build
+on: [workflow_dispatch, pull_request]
+
+jobs:
+  build:
+    name: Build
+    strategy:
+      matrix:
+        os: [large-ubuntu-22.04-xxl]
```

Comment:
We don't really need the paid fast runners, if caching is working properly.  So ubuntu-22.04 or what it is called, should be fine.  Furthermore, the matrix build is not necessary here, if we only use one platform anyway